packages/components/pie-text-input/README.md

```diff
@@ -19,23 +19,25 @@
 7. [Slots](#slots)
 8. [Contributing](#contributing)
 
-## pie-input
+## pie-text-input
 
-`pie-input` is a Web Component built using the Lit library.
+`pie-text-input` is a Web Component built using the Lit library.
 
 This component can be easily integrated into various frontend frameworks and customized through a set of properties.
 
 
 ## Installation
 
-To install `pie-input` in your application, run the following on your command line:
+To install `pie-text-input` in your application, run the following on your command line:
+
+_Note: Older versions of this component were named `pie-input`._
```

Comment:
Nice addition, I think including the version number would be helpful though.
```suggestion
_Note: Versions of this component prior to v0.19.0 were named `pie-input`._
```

Reply:
I agree but I thought we can't tell what version it'll be until its merged as there are other PRs that also raise the version number

Reply:
i.e. it might end up being versions prior `0.19.2`